common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -82,4 +84,8 @@ public static Map<String, String> getParameterMap(Map<String, ?> parameterObjs)
         }
         return parameters;
     }
+
+    public static String xContentBuilderToString(XContentBuilder builder) {
```

Comment:
I believe we don't need this method.

Reply:
XContentBuilder already supports `toString`.